### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

**Comment by DeathlyPlays** [comment]
10/10 titling from sleepy mobile Insist right there

File: showdown.js [hunk 1]
```diff
@@ -108,31 +108,31 @@ Showdown.prototype.finalize = function finalize(parts) {
                 challengekeyid: id,
                 challenge: str,
                 name: nickname,
-                pass: password
+                pass: password,
             }
         },
-        function finish(error, response, body) {
```
Comment (sugar700) [inline showdown.js:114]: Does that work with removed parameters?

**Comment by sugar700** [comment]
Just curious, do you want to integrate this with Pokemon Showdown? I understand the intent to change code style to match Showdown style, however this isn't strictly speaking a part of Pokemon Showdown and as such doesn't have to follow its style (and I will admit, I like not using semicolons in JavaScript myself).

That said, there are some things I would love to see changed in PSDevBot in event you want to contribute somehow, but I'm too busy to change them because they are low priority, in particular modernization of this code to ECMAScript 6. To be exact, PSDevBot could be improved by means of template literals usage, `let`/`const`, request promises, use of maps instead of objects. It also would be neat to have some unit testing to ensure things keep working, which I may as well create soon I suppose.

File: showdown.js (hunk 1 above)
Comment (DeathlyPlays) [reply showdown.js:114]: It should, that parameter was not used

**Comment by DeathlyPlays** [comment]
Sure I could do let/const things

**Comment by DeathlyPlays** [comment]
If you want we could try integrating it with PS

**Pull request closed without merging**